.pre-commit-config.yaml

```diff
@@ -63,6 +63,11 @@ repos:
     -   id: sort-all
         types: [file, python]
 
+-   repo: https://github.com/abravalheri/validate-pyproject
+    rev: "v0.16"
+    hooks:
+    - id: validate-pyproject
+
```

Comment:
This check is currently skipping due to the selected files:

![image](https://github.com/SciTools/iris-grib/assets/40734014/e6f2c17d-5965-4271-92f6-1c41704d6cc9)

https://github.com/SciTools/iris-grib/blob/adc4a412825e2e85a90337d22983c60e57413f34/.pre-commit-config.yaml#L4-L11